sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/ConversationAnalysisClientLiveTests.cs

```diff
@@ -45,7 +46,8 @@ public async Task AnalyzeConversation()
             Assert.IsNotNull(response);
 
             // deserialize
-            dynamic conversationalTaskResult = response.Content.ToDynamicFromJson();
+            DynamicDataOptions options = new() { CaseMapping = DynamicCaseMapping.PascalToCamel };
```

Comment:
@annelo-msft would you like me to change the tests (the samples should be fine already, IIRC) to match our defaults in case customers look at them? I could do that in a separate PR so it's not a bother for you if so.

Reply:
I had understood that this is what we want to do in customer-facing documentation.  What is it that makes you think this should change?

Reply:
Cool.  As I understand the vision for this type, we want it -- and our docs -- to look as much like idiomatic .NET code as possible.  Given that, I feel it's regrettable that the identified risks resulted in needing to pass options into the type in every instance in order to use it in the intended way -- I wish we had been able to find a solution that improved the customer experience here.  But I also know there are diverging perspectives on how customers will want to use the type, and we have made it possible for them to make an informed decision based on their own preferences.

Reply:
Updated to simplify a bit.